Fakebook.Profile/Fakebook.Profile.UnitTests/APITests/ProfileIntegrationTests.cs

```diff
@@ -22,8 +22,9 @@ public async Task ProfileRepoAddProfile()
 
             var repo = new ProfileRepository(context);
             await repo.CreateProfileAsync(test);
+            int lastId = context.EntityProfiles.Count();
 
-            var compare = context.EntityProfiles.Local.Single(x => x.Id == 3); //ID is 3 because there's already 2 users in seed data
+            var compare = context.EntityProfiles.Local.Single(x => x.Id == lastId);
```

Comment:
this is still a little brittle, since nothing's forcing us to make the seed data IDs sequential. Max + 1 is a more robust solution. but you can leave it as is if you want

Reply:
Leaving it as is for now.